conftest.py

```diff
@@ -0,0 +1,121 @@
+import json
+import os
+
+import boto3
+import pytest
+from moto import mock_iam, mock_s3
+
+
+@pytest.fixture(scope="function")
```

Comment:
"function" is the default fixture scope so this isn't really necessary. I would either include it in every fixture if you want to be explicit, or (probably better) not include it in any, otherwise it gives the impression in this code that there's a difference between the fixtures with it and the fixtures without.